buttercup/cogs/rules.py

```diff
@@ -22,6 +36,24 @@ def extract_sub_name(subreddit: str) -> str:
     return subreddit
 
 
+def contains_any(text: Optional[str], keywords: List[str]) -> bool:
+    """Determine if the text contains any of the keywords."""
+    if text is None:
+        return False
+    lower_text = text.casefold()
+    for word in keywords:

```

Comment:
The "pythonic" way of doing these types of iterations is mostly using list comprehensions, e.g:
```
return any(lower_text in word for word in keywords)
```